src/specklepy/objects/geometry/__init__.py

```diff
@@ -5,6 +5,14 @@
 from specklepy.objects.geometry.point import Point
 from specklepy.objects.geometry.polyline import Polyline
 from specklepy.objects.geometry.vector import Vector
+from specklepy.objects.geometry.box import Box
```

Comment:
in these cases, its fine to use relative imports, like `from .point import Point`

Reply:
done 👍🏻